.github/workflows/helm_lint.yml

```diff
@@ -0,0 +1,53 @@
+name: Helm Lint and Test Charts
+
+on: pull_request
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
```

Comment:
One quick thing, on the action itself, we use [`dorny/paths-filter`](https://github.com/dorny/paths-filter) [in our other actions](https://github.com/cfpb/consumerfinance.gov/blob/main/.github/workflows/backend.yml#L41-L50) in our other actions to make it fast-pass if there are no changes to the relevant files. Would that be useful here?